README.md

```diff
@@ -678,7 +678,9 @@ The encrypted binary will have the following structure:
 - Padding to ensure the encrypted length is a multiple of 4 words
 - Signature metadata block
 
-The AES key must be provided as a .bin file of the 256 bit AES key to be used for encryption.
+The AES key must be provided as a 1024 bit .bin file containing 4-way shares of each word of the 256 bit AES key to be used for encryption.
```

Comment:
I'm very confused as a user what I'm actually supposed to do. should i pass a file of length 128?

Reply:
You need to pass a file of length 1024 bits, or 128 bytes - I went with bits, because you normally talk about key length in bits rather than bytes

Reply:
Might be worth mentioning the length in both bits **and** byes? Since as William points out key-lengths are measured in bits, but the `ls` command only shows file-sizes in bytes.